quark/subproject.py

```diff
@@ -239,6 +258,19 @@ def set_local_ignores(self, subprojects_dir, modules):
         pass
 
 class GitSubproject(Subproject):
+    @staticmethod
+    def get_version_control():
+        return "git"
+
+    def get_env_variables(self, toplevel):
+        return {
+            **super().get_env_variables(toplevel=toplevel),
+            **{
+                "sha1": str(self.ref),
+                "version_control": self.get_version_control()
```

Comment:
Can we move "version_control" in `get_env_variables` if the API `get_version_control()` is there to stay, or simply use "git" / "svn" if there is no more need for it?

Reply:
Changed in https://github.com/comelz/quark/pull/27/commits/ee6e41ed5393e010b8dd24de62bbfe021cc50f09